test/e2e.go

```diff
@@ -57,6 +53,17 @@ func newClients(t *testing.T) (*clients, error) {
 	}, nil
 }
 
+func getKubeConfig() string {
+	config, found := os.LookupEnv("KUBECONFIG")
+	if found {
+		return config
+	}
```

Comment:
```suggestion
	if config, found := os.LookupEnv("KUBECONFIG"); found {
		return config
	}
```
just to match what you do below